### PR TITLE
Updated OpenShift Master iptables rules

### PR DESCRIPTION
**Pull request by mtnbikenc**
* Removed unneeded rules
* Moved etcd rule to conditional based on usage of embedded etcd

https://bugzilla.redhat.com/show_bug.cgi?id=1386329

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: playbooks/common/openshift-master/config.yml [hunk 1]
```diff
@@ -166,16 +164,11 @@
...
-      port: 2224/tcp
-    - service: Corosync UDP
-      port: 5404/udp
-    - service: Corosync UDP
-      port: 5405/udp
+  - role: os_firewall
+    os_firewall_allow:
+    - service: etcd embedded
+      port: 4001/tcp
+    when: embedded_etcd | bool
```
Comment (abutcher) [inline playbooks/common/openshift-master/config.yml]: We could check this as `groups.oo_etcd_to_config | default([]) | length == 0`. There's also an `openshift.master.embedded_etcd` fact which is equivalent and arrived at similarly.

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
@abutcher thanks for the tip on the var
@detiber All the tests look good now

**Pull request merged**